src/praisonai-agents/praisonaiagents/skills/manager.py

```
@@ -243,6 +243,315 @@ def load_resources(self, name: str) -> bool:
+                return {"success": False, "error": f"Cannot delete skill '{name}' - no path available"}
+            
+            # Remove from memory first
+            del self._skills[name]
+            
+            # Remove directory
+            import shutil
+            shutil.rmtree(skill.properties.path)
+            
+            return {"success": True, "skill": name, "path": str(skill.properties.path)}
+            
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)